src/data/machines.js

```diff
@@ -41,6 +41,10 @@ export const monthlyStoragePrice = 0.04 // from https://cloud.google.com/compute
 export const storagePrice = monthlyStoragePrice / 730 // per GB hour using 730 hours per month from https://cloud.google.com/compute/pricing
 export const dataprocCpuPrice = 0.01 // dataproc costs $0.01 per cpu per hour
 
+// TODO calculate cost based on app, rather than constant
+// From https://cloud.google.com/compute/all-pricing#sustained_use monthly pricing
+export const appPrice = 194.18 // per GB hour using 730 hours per month from https://cloud.google.com/compute/pricing
```

Comment:
I don't think that we're expecting people to leave their clusters up for the whole month, so they probably won't get the full sustained use discount. It might be better to show the base hourly rate, since that's the worst case estimate.

Thoughts?

Reply:
It would worry me a bit to use a different calculation method than we are using for runtimes. Do runtimes also have a sustained use cost model? (I want to try to be consistent, cause my guess is the use case is consistent, but I am not sure. Maybe a product question?)

Reply:
Runtimes do too, but we use the base hourly cost. Note that this is currently doing the reverse of the calculation we do for runtimes. It's only for disks that we use the monthly and divide by 730.

(In case it wasn't clear, google's price sheet includes the discount when showing monthly price, but not for hourly)

Reply:
Now that we're computing the cost directly, this code could be removed.